test/conformance/usm/urUSMDeviceAlloc.cpp

```diff
@@ -27,16 +28,20 @@ struct urUSMDeviceAllocTest : uur::urQueueTestWithParam<uur::BoolTestParam> {
             ASSERT_SUCCESS(urUSMPoolRelease(pool));
         }
         UUR_RETURN_ON_FATAL_FAILURE(
-            uur::urQueueTestWithParam<uur::BoolTestParam>::TearDown());
+            uur::urQueueTestWithParam<uur::USMDeviceAllocParams>::TearDown());
     }
 
     ur_usm_pool_handle_t pool = nullptr;
+    bool usePool = std::get<0>(getParam()).value;
 };
 
+// The 0 value parameters are not relevant for urUSMDeviceAllocTest tests, they are used below in urUSMDeviceAllocAlignmentTest for allocation size and alignment values
```

Comment:
looks like formatting missed these comments

Reply:
Trying to format this locally and I get no change, should it be done in a particular way?

Reply:
ah right, I guess just split the line so it's under the 80 column maximum width

Reply:
Cool updated it now